app/assets/javascripts/thyme.js

```diff
@@ -0,0 +1,1092 @@
+/* a boolean that helps to deactivate all key listeners
+   for the time the annotation modal opens and the user
+   has to write text into the command box */
+var lockKeyListeners = false;
+
+/* when callig the updateMarkers() method this will be used to save an
+   array containing all annotations */
+var annotations = null;
+
+/* helps to find the annotation that is currently shown in the
+   annotation area */
+var activeAnnotationId = 0;
+
+/* if the window width (in px) gets below this threshold value, hide the control bar
+   (default value) */
+var hideControlBarThreshold = {
+  x: 850,
+  y: 500
+};
+
+// return the start time of the next chapter relative to a given time in seconds
+nextChapterStart = function(seconds) {
+  var chapters = document.getElementById('chapters');
+  var times = JSON.parse(chapters.dataset.times);
+  if (times.length == 0) {
+    return;
+  }
+  var i = 0;
+  while (i < times.length) {
+    if (times[i] > seconds) {
+      return times[i];
+    }
+    ++i;
+  }
```

Comment:
Use case for a `for` loop.

Reply:
No sorry, this was already the case beforehand in the coffeescript. We only consider new changes here, so ignore this. See the review comment for a proposal.